grails5 upgrade - geb issue

niravassar opened the pull request.

Commit:
grails5 upgrade

sdelamo commented on complete/build.gradle:
this should be decoupled from the grails version

sdelamo commented on complete/build.gradle:
this  should be 2.4

sdelamo commented on complete/build.gradle:
this should use the same version as grails_i18n guide

sdelamo commented on complete/build.gradle:
this should use the same version as grails_i18n guide

sdelamo commented on complete/gradle.properties:
5.0.1

sdelamo commented on complete/gradle.properties:
7.1.0

sdelamo commented on complete/gradle.properties:
this should be the same as grails_i18n guide

sdelamo commented on complete/gradle.properties:
this should be 2.4

sdelamo commented on complete/gradle.properties:
this should be the same as grails_i18n guide

sdelamo commented on initial/build.gradle:
this should be decoupled from the grails version

sdelamo commented on initial/build.gradle:
this should be 2.4

sdelamo commented on initial/gradle.properties:
this should be 5.0.1

sdelamo commented on initial/gradle.properties:
this should be 7.1.0

sdelamo commented on initial/gradle.properties:
this should be the same as grails_i18n version

sdelamo commented on initial/gradle.properties:
this should be the same as grails_i18n version

jamesfredley committed:
Update README to point at canonical guide on grails.apache.org (refs apache/grails-static-website#354)

jamesfredley committed:
Replace dead publisher CI with sample-app gradle check workflow

Delete: githubactions-build.sh
Add: .github/workflows/gradle.yml (auto-detects Java from Gradle wrapper version)

(refs apache/grails-static-website#354)
